openmodelica/linearization/simNonlinear_numeric.mos

```diff
@@ -0,0 +1,78 @@
+// name:     nonlinsys numeric linearization
+// keywords: nonlinsys, linearization
+// status:   correct
+// teardown_command: rm -rf p.nonlinsys.* p.nonlinsys_* output.log linear_p_nonlinsys.* linear_p.nonlinsys.mo linear_p_nonlinsys_* linear_p_nonlinsys p.nonlinsys
+// 
+//  Case for numeric linearization of nonlinsys model
+//
+loadFile("modelnonlinsys.mo");
+//linearize with numeric linearization and data recovery
+//linearize(p.nonlinsys, stopTime=0);
+simulate(p.nonlinsys, stopTime=0, simflags="-l=0 -l_datarec");
+getErrorString();
+loadFile("linear_p.nonlinsys.mo");
+list(linear_p_nonlinsys);
+simulate(linear_p_nonlinsys);
+getErrorString();
+readFile("linear_p_nonlinsys.log"); // Check that output log is emtpy
+
+// Result:
+// true
+// record SimulationResult
+//     resultFile = "S:/Scratch/p.nonlinsys_res.mat",
```

Comment:
The test suit is unstable due to this and a similar line further down. Just replace it with `resultFile = "p.nonlinsys_res.mat",`.